lms/startup.py

```diff
@@ -94,7 +94,7 @@ def enable_stanford_theme():
     theme_root = settings.ENV_ROOT / "themes" / settings.THEME_NAME
 
     # Include the theme's templates in the template search paths
-    settings.TEMPLATE_DIRS.insert(0, theme_root / 'templates')
+    settings.TEMPLATES[0]['DIRS'].insert(0, theme_root / 'templates')
```

Comment:
I agree with its fragility. I'm going to attempt creating a `DEFAULT_TEMPLATE_ENGINE` setting pointing at `TEMPLATES[0]` and use that instead everywhere.